CHANGES.txt

```diff
@@ -1,3 +1,5 @@
+Add proper hitogram checkup to fix flaky failThresholdSinglePartition test
```

Comment:
not proper format, I would personally remove and leave to committer

If you want it correctly (which always conflicts on merge so committer has to fix... hence why I always exclude... but thats me), you would put this under `4.1` (if its going in the 4.1 release), put it at the top and add ` * ` prefix

Reply:
> not proper format, I would personally remove and leave to committer

Agree. 

And... this patch is to fix test only. So we will not add an entry in the `CHANGES.txt`.